internal/usecase/toggle_todo_test.go

```diff
@@ -27,7 +26,7 @@ func TestToggleTodoUsecase(t *testing.T) {
 	if !isToggleTodoMockCalled {
```

Comment:
Why do we need to check isToggleTodoMockCalled ? We're checking isUpdateTodoMockCalled right?